crates/astria-conductor/src/executor/mod.rs

```diff
@@ -375,40 +377,36 @@ impl Executor {
     /// - if execution or finalization of a block from celestia fails
     async fn handle_executor_command(&mut self, cmd: Option<ExecutorCommand>) -> eyre::Result<()> {
         let Some(cmd) = cmd else {
-            bail!("cmd channel closed unexpectedly; shutting down")
+            bail!("cmd channel closed unexpectedly");
         };
 
         match cmd {
             ExecutorCommand::FromSequencer {
                 block,
             } => {
-                let height = block.header().height.value();
+                let height = block.header().height;
                 let block_subset =
                     SequencerBlockSubset::from_sequencer_block(*block, self.rollup_id);
 
                 match self.execute_block(block_subset).await {
                     Ok(executed_block) => {
                         if let Err(e) = self.update_soft_commitment(executed_block.clone()).await {
-                            let error: &(dyn std::error::Error) = e.as_ref();
-                            error!(height = height, error, "failed to update soft commitment");
+                            let error: &dyn std::error::Error = e.as_ref();
+                            warn!(%height, error, "failed to update soft commitment");
```

Comment:
I think failing to update the soft committment should be treated as an error. Something has gone wrong, the state of the rollup is not correct.

Reply:
The event levels are conceptually:

1. Warn: an error occured, but we can handle it (even if just reporting), and then continue working
2. Error: an error occured and the service cannot handle it. This is usually followed by a shutdown.

So the question here then is: can we go back to a valid state after this error occurred?

From the sound of it we can't - then the logical next step is not to emit an event but stop the service.

Reply:
Agreed, I don't believe this is reconcilable. Stopping the service sounds reasonable.